### ISSUE TITLE
Animations don't work with @apply

### ISSUE THREAD

**Issue by itpropro** [windicss/windicss#157]
My animation classes don't work when used in an @apply statement:
Works:

```html
<template>
  <div class="relative w-40 h-40 rounded-full bg-teal-500 opacity-60 animate-pulse"></div>
</template>
```

Doesn't work:

```html
<template>
  <div class="pulse-class"></div>
</template>

<style scoped>
.pulse-class {
  @apply relative w-40 h-40 rounded-full bg-teal-500 opacity-60 animate-pulse;
}
</style>
```

**Comment by itpropro** [comment]
Thanks for creating a fix so fast, unfortunately, with the above example, it's still the same behavior. I updated all packages and made sure that windicss 2.5.0 is installed. Here is the difference in generated CSS:
Working (no @apply):
![image](https://user-images.githubusercontent.com/15030951/111600256-84271300-87d1-11eb-862f-85a81a6a5abd.png)
Not working (with @apply):
![image](https://user-images.githubusercontent.com/15030951/111600289-8d17e480-87d1-11eb-9096-f4791471cbaf.png)

Jsut for reference, my package.json:

```json
{
  "name": "nuxt-playground",
  "version": "1.0.0",
  "private": true,
  "scripts": {
    "dev": "nuxt",
    "build": "nuxt build",
    "start": "nuxt start",
    "generate": "nuxt generate",
    "lint:js": "eslint --ext \".js,.vue\" --ignore-path .gitignore .",
    "lint": "yarn lint:js"
  },
  "dependencies": {
    "core-js": "^3.8.3",
    "nuxt": "^2.14.12"
  },
  "devDependencies": {
    "@nuxtjs/composition-api": "^0.22.1",
    "@nuxtjs/eslint-config": "^5.0.0",
    "@nuxtjs/eslint-module": "^3.0.2",
    "@vue/composition-api": "^1.0.0-rc.5",
    "@windicss/plugin-interaction-variants": "^1.0.0",
    "@windicss/plugin-scrollbar": "^1.2.3",
    "babel-eslint": "^10.1.0",
    "eslint": "^7.18.0",
    "eslint-config-prettier": "^7.2.0",
    "eslint-plugin-nuxt": "^2.0.0",
    "eslint-plugin-prettier": "^3.3.1",
    "eslint-plugin-vue": "^7.5.0",
    "prettier": "^2.2.1",
    "vue-windicss-preprocess": "^2.2.0"
  }
}
```

**Comment by voorjaar** [comment]
Yeah. Sorry about this. Can you provide me a copy of the completed css?

**Comment by voorjaar** [comment]
Oh. By the way. Can you switch your project to https://github.com/windicss/windicss-webpack-plugin. We are planning to give up support for vue-windicss-preprocess.

**Comment by hannoeru** [comment]
@voorjaar @itpropro I think is [nuxt-windicss-module](https://github.com/windicss/nuxt-windicss-module).

**Comment by voorjaar** [comment]
Temporarily close, please switch to [nuxt-windicss-module](https://github.com/windicss/nuxt-windicss-module).

**Comment by itpropro** [comment]
Thanks for mentioning the webpack module, I will stay with that.
I was not able to get the nuxt-windicss-module to work without vite at all, wich is why I used the `vue-windicss-preprocess` one from the documentation. 
The simplest reproduction that I did was creating a brand new Nuxt project with npx and just adding the `nuxt-windicss-module` module. It just starts to work when nuxt-vite is installed.
See https://github.com/windicss/nuxt-windicss-module/issues/25

**Comment by voorjaar** [comment]
Yeah. Thank you. We will help you to fix it.

**Comment by itpropro** [comment]
I got nuxt-windicss to work (beside the current not working version 0.4.5) and the problem still persists. You can take a look at the code at https://github.com/itpropro/nuxt-windicss, I added a Animation.vue component to recreate the bahavior.
You can just do `yarn` and `yarn dev` to see the issue. Currently you probably have to pin the nuxt-windicss version to 0.4.4, because 0.4.5 ist broken (https://github.com/windicss/nuxt-windicss-module/issues/28).

Let me know, if I can do anything else to support 😃

**Comment by harlan-zw** [comment]
Hey @itpropro 

This seems to be working correctly with `"nuxt-windicss": "^0.4.6",`, please update.

**Comment by itpropro** [comment]
Hey,
Unfortunately, the bug is still there, I have updated my example repository to reflect that both green circles should be pulsing. The left one is with @apply and the right one without.
@voorjaar can you please reopen the issue?
